crates/interpreter/src/interpreter/memory.rs

```diff
@@ -67,6 +67,14 @@ impl Memory {
         &self.data[offset..offset + size]
     }
 
+    /// Memory copy
+    /// Copies memory inplace to a returns a [u8]
+    #[inline(always)]
+    pub fn copy(&self, offset: usize, size: usize) -> [u8; 32] {
+        let mut copy = [0u8; 32];
```

Comment:
This looks like `copy_word` because `copy` is only `32`, it should be `size`

Reply:
addressed this with returning a vec of bytes and then passing as a slice to the data